app-editors/qhexedit2/qhexedit2-0.8.4.ebuild

```diff
@@ -20,8 +20,8 @@ IUSE="doc gui python"
 REQUIRED_USE="python? ( ${PYTHON_REQUIRED_USE} )"
 
 PATCHES=(
-	"${FILESDIR}/${P}-fixtest.patch"
-	"${FILESDIR}/${P}-setup.py.patch"
+	"${FILESDIR}/${PN}-fixtest.patch"
```

Comment:
why did you remove the version string? In general `${P}-blabla.patch` is preferred

Reply:
cause patches used both qhexedit2-0.8.4.ebuild and qhexedit2-0.8.4_p20170719.ebuild

Reply:
then make it `qhexedit2-0.8.4-fixtest.patch` and do `"${FILESDIR}/${PN}-0.8.4-fixtest.patch"` in the qhexedit2-0.8.4_p20170719.ebuild